src/main/resources/com/hcl/appscan/jenkins/plugin/Messages.properties

```diff
@@ -33,6 +33,9 @@ warning.asoc.certificates = Allowing Untrusted Connections is not applied to thi
 error.application.empty.ui = Application list is empty. Please check the credentials for service connection. 
 error.url.validation.ui = Unless this is a private network, please verify your domain.
 error.url.validation = An error occurred while validating the Starting URL: {0}.
+error.dast.subscription.validation= You don't have a valid DAST subscription.
+error.sast.subscription.validation= You don't have a valid SAST subscription.
+error.sca.subscription.validation= You don't have a valid SCA subscription.
```

Comment:
I suggest combining these 3 strings into a single string that takes an input. For example:
`error.subscription.invalid=You don't have a valid {0} subscription.`

Reply:
Sure, I will use only 2 statements, one for the backend validation message & other for the UI.